syntax = "proto3";

message MxMcp23008Store {
    repeated bool state = 1;
}